android/app/src/main/java/org/fossasia/openevent/utils/Notification.java

```diff
@@ -9,65 +9,31 @@
 import android.media.RingtoneManager;
 import android.net.ConnectivityManager;
 import android.net.Uri;
-import android.support.design.widget.Snackbar;
 import android.support.v4.app.NotificationCompat;
-import android.support.v4.widget.SwipeRefreshLayout;
-import android.view.View;
 
 import org.fossasia.openevent.R;
 
-import timber.log.Timber;
-
 import static android.content.Context.NOTIFICATION_SERVICE;
 
 /**
  * Created by vishwesh3 on 8/12/16.
  */
 
-public abstract class ShowNotificationSnackBar {
-    private  Context context;
-    private View view;
-    private SwipeRefreshLayout swipeRefreshLayout;
-    private Snackbar snackbar;
-    public ShowNotificationSnackBar(Context context, View view, SwipeRefreshLayout swipeRefreshLayout) {
-        this.context =context;
-        this.view = view;
-        this.swipeRefreshLayout = swipeRefreshLayout;
-        buildNotification();
-        showSnackBar();
-    }
-
-    public abstract void refreshClicked();
+public class Notification {
 
-    public void showSnackBar(){
-        if (view == null) {
-            if (swipeRefreshLayout != null) {
-                view = swipeRefreshLayout;
-            } else {
-                Timber.e("Error getting suitable parent while showing snackbar");
-                return;
-            }
-        }
+    private Context context;
 
-        snackbar = Snackbar.make(view, R.string.waiting_for_network, Snackbar.LENGTH_INDEFINITE)
-                .setAction(R.string.snackbar_refresh_action, v -> {
-                    if (swipeRefreshLayout!=null)
-                        swipeRefreshLayout.setRefreshing(true);
-                    snackbar.dismiss();
-                    refreshClicked();
-                });
-        snackbar.show();
+    public Notification(Context context) {
+        this.context = context;
     }
 
-
-
-    public void buildNotification(){
-        if(context == null)
+    public void buildNotification() {
+        if (context == null)
             return;
 
         ConnectivityManager connectivityManager = (ConnectivityManager) context.getSystemService(Context.CONNECTIVITY_SERVICE);
         android.net.NetworkInfo isWifi = connectivityManager.getNetworkInfo(ConnectivityManager.TYPE_WIFI);
```

Comment:
change the field name `isWifi` to something else, it seems a boolean by name. may be just `Wifi` is enough.